torch/lib/c10d/CMakeLists.txt

```
@@ -45,15 +45,16 @@ endfunction()
 
 set(C10D_SRCS
   FileStore.cpp
-  HashStore.cpp
   ProcessGroup.cpp
-  ProcessGroupRoundRobin.cpp
   Store.cpp
   PrefixStore.cpp
-  TCPStore.cpp
   Utils.cpp
   )
 
+if(NOT MSVC)
```

Please use platform rather than compiler specific check
```suggestion
if(NOT WIN32)
```